packages/validator/src/services/indices.ts

```diff
@@ -101,7 +101,13 @@ export class IndicesService {
     ApiError.assert(res, "Can not fetch state validators from beacon node");
 
     const newIndices = [];
+
+    const allValidators = new Map<ValidatorStatus, ValidatorResponse[]>();
+
     for (const validatorState of res.response.data) {
+      // Group all validators by status
+      allValidators.set(validatorState.status, [validatorState]);
```

Comment:
This set will overwrite previous sets when validators share a status.

Reply:
Good catch! Thanks